cmd/bootstrap.go

```diff
@@ -101,6 +103,10 @@ func setConfig(ctx context.Context, configPath string) (*config, error) {
 			PrivateKey:  "",
 			ListenAddr:  "/ip4/0.0.0.0/tcp/7999",
 			BiddingWait: 10 * time.Second,
+			RPCConfig: rpcClientConfig{
+				Timeout: 5 * time.Minute,
```

Comment:
need to confirm this is the right timeout config -- 5 minutes for total server request time before timing out?